### DISCUSSION TITLE
equation环境中的字体可以更改吗？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#123] [Q&A]
你好，我发现我们equation中的数字字体有点不一样：
这是我们模板的：
![Screenshot 2022-04-15 183028](https://user-images.githubusercontent.com/45897471/163560784-b97caa10-d36f-43fb-a261-c68ebc14f177.jpg)
这是texworks默认的：
![Screenshot 2022-04-15 183117](https://user-images.githubusercontent.com/45897471/163560871-56cc0ec1-44dd-4f82-a02f-5f78a08e96b4.jpg)

**Comment by sikouhjw** [comment]
https://github.com/whutug/whu-thesis/blob/a0cb5918638fece6c7fa171bb6be179dcf6a3a33/whu-thesis.cls#L478-L481
请阅读模板使用手册后再使用模板。

**Reply by earthmoon** [reply]
好吧，等以后看能不能学会看代码。话说你这个直接引用具体代码的回复是怎么做到的？

**Reply by sikouhjw** [reply]
https://stackoverflow.com/questions/23821235/how-to-link-to-specific-line-number-on-github

**Reply by earthmoon** [reply]
好的，谢谢！
